python_backend/main.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+from fastapi import FastAPI, Response, status, HTTPException
+from fastapi.params import Body
+from pydantic import BaseModel
+from random import randrange
+from dotenv import dotenv_values
+import psycopg
+import time
+
+app = FastAPI()
+secrets = dotenv_values(".env")
+
+class Login(BaseModel):
```

Comment:
Move classes to their own file to be more organized and have a central folder of all classes/models